plugin/selection_range.py

```diff
@@ -25,31 +25,38 @@ def run(self, edit: sublime.Edit, event: Optional[dict] = None) -> None:
             self._regions.extend(self.view.sel())
             self._change_count = self.view.change_count()
             session.send_request(Request(self.method, params), self.on_result, self.on_error)
+        else:
+            self._run_builtin_expand_selection("No {} found".format(self.capability))
```

Comment:
This message is quite internal to the protocol. I feel user shouldn't need to see it.

Reply:
Actually I think users have to get comfortable with fooProvider and barProvider and such as they'll end up in the contexts of keybindings eventually after #1069 (unless we go with #1093) (or we can do both).